src/index.html

```diff
@@ -10,13 +10,64 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
-    <title>Moyo header</title>
+    <title>MOYO/Header</title>
     <link
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="nav-logo">
+        <a href="homepage">
+          <img
+            src="images\logo.png"
+            alt="MOYO-logo"
+          />
+        </a>
+      </div>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav-item is-active apple-active">
+            <a
+              href="APPLE"
+              class="is-active"
+            >
+              APPLE
+            </a>
+          </li>
+          <li class="nav-item">
+            <a href="SAMSUNG">SAMSUNG</a>
+          </li>
+          <li class="nav-item">
+            <a href="SMARTPHONES">SMARTPHONES</a>
+          </li>
+          <li class="nav-item">
+            <a
+              href="LAPTOPS & COMPUTERS"
+              data-qa="hover"
+            >
+              LAPTOPS & COMPUTERS
+            </a>
+          </li>
+          <li class="nav-item">
+            <a href="GADGETS">GADGETS</a>
+          </li>
+          <li class="nav-item">
+            <a href="TABLETS">TABLETS</a>
```

Comment:
The `href` attribute should not contain spaces and should start with the `#` symbol if it is an anchor link.